test/checkbox.test.tsx

```diff
@@ -0,0 +1,102 @@
+import { test, Page } from "bigtest";
+import { Body, CheckBox as Interactor } from "../src/index";
+import { Checkbox, FormControlLabel } from "@material-ui/core";
+import { render } from "./helpers";
+
+export default test("Checkbox")
+  .step(Page.visit("/"))
+  .child("filter", (test) =>
+    test
+      .child("by locator", (test) =>
+        test
+          .step("render", () => render(<FormControlLabel label="locator" control={<Checkbox />} />))
+          .assertion(Interactor("locator").exists())
```

Comment:
Locator is loaded in this context. We are technically testing that the `checkbox` locator works under the hood but it works based on the labels for the checkbox. So see by locator as the test name, and then locator in the label tripped me for a second. Should the child test name be something like `find checkbox by label`? And then the label be something like `Send emails` or something for example?

Reply:
Just a comment not a blocker

Reply:
Ok, I'll fix it.